packages/myst-templates/src/template.ts

```diff
@@ -65,8 +65,15 @@ class MystTemplate {
     if (!fs.existsSync(templateYmlPath)) {
       throw new Error(`The template yml at "${templateYmlPath}" does not exist`);
     }
+
+    interface TemplateYaml {
+      files: string[];
+      [key: string]: any;
+    };
     const content = fs.readFileSync(templateYmlPath).toString();
-    return yaml.load(content);
+    const yamlData : TemplateYaml = yaml.load(content) as TemplateYaml;
```

Comment:
I think this is the lint failure:

```suggestion
    const yamlData: TemplateYaml = yaml.load(content) as TemplateYaml;
```

Reply:
Heh, so if you run `npm run format` and then `npm run lint`, you get 12 errors! So doesn't look like eslint and prettier are configured to work together correctly.

Reply:
CI is checking `npm run lint` and `npm run lint:format`. They should be working locally and do work together... ?!

I am using vscode with eslint/prettier default extensions, that formats on save.

Reply:
Would you be open to a git hook to achieve something similar?